examples/README.md

```diff
@@ -12,8 +12,93 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Examples
+# Configuration
+**Redirect all requests to "www"-subdomain:**  
+*example.com -> www.example.com*
+```
+txtdirect {
+  enable www
+}
+```
 
-No examples are available yet, but we are happy to get new contributions.
+**Redirect to host provided in TXT record:**  
+*Default: Return 404 on empty record*
+```
+txtdirect {
+  enable host
+}
+```
+
+**Redirect to host provided in TXT record:**  
+*Default: Redirect to "www"-subdomain on empty record*
+```
+txtdirect {
+  enable host www
+}
+```
+
+**Redirect to host provided in TXT record:**  
+*Default: Redirect to "www"-subdomain on empty record*
+```
+txtdirect {
+  redirect https://example.com
```

Comment:
This example doesn't use TXT records at all; change to something like this ?
```diff
-**Redirect to host provided in TXT record:**
+**Redirect to host provided after redirect keyword**
```

Reply:
But it would redirect a host record and only default to the redirect, if no record is used right?

Reply:
I'm not sure, if host isn't enabled (https://github.com/txtdirect/txtdirect/blob/master/txtdirect.go#L156)

Reply:
I forgot about that line, my bad https://github.com/txtdirect/txtdirect/blob/master/caddy/caddy.go#L72

Reply:
So with the fixes lgtm?